### DISCUSSION TITLE
МГТС конфиг

### DISCUSSION THREAD

**Discussion by ghost** [dovecoteescapee/ByeDPIAndroid#162] [General]
Прошу помочь с конфигом для МГТС.
Пробовал конфиги из последних тем - ничего не помогает

**Comment by PinkLine** [comment]
Настройки -> Use command line settings - on (в разделе ByeDPI), далее открываешь command line editor и в нем пишешь "-s1 -o1 -Ar -o1 -At - f-1 -r1+s -As"без кавычек

**Reply by ghost** [reply]
работает! спасибо!

**Reply by X-RayGuy** [reply]
работает на мгтс москва,но проблема иногда зависает и не грузит нормально с большими задержками.
